server/config/config.go

```diff
@@ -199,6 +199,7 @@ type StateSyncConfig struct {
 	SnapshotKeepRecent uint32 `mapstructure:"snapshot-keep-recent"`
 }
 
+// Deprecated: Use StreamingConfig instead
```

Comment:
This doesn't actually deprecate anything. The only way anyone would know that this stuff is deprecated is if they're looking directly at this file. Is that intended?

It looks like this stuff is all ignored now. If that's the case, it should be removed (as well as its stuff in the toml).

Reply:
I had that comment to remind me. Looks like I forgot to remove it. It doesn't exist in upstream. I wasn't certain if that needed to be removed or how to notify deprecation. This may need to be a separate ticket upstream to remove this after a merge.

Reply:
But it's not deprecated. Deprecated means its still usable, but that something else is replacing it. But it's not still usable. That stuff is completely ignored.

Reply:
You're right. They were added in between ADR proposal (the original and mine) to fix an issue with the store for the file streamer, which no longer exists. They're ignored by the plugin system. The best course is to remove them. They can be added manually if needed.

Reply:
I see that this comment was removed. The rest of my statements still apply though. There's nothing left that uses the old `StoreConfig`. It needs to be removed along with the stuff it contains and the config file section for it.

Grepping the repo for "streamers" only returns results in this `config.go`, `toml.go`, and `config_test.go`. That `StoreConfig` is only used in `config.go` and `config_test.go` (but could easily be confused with the `StoreConfig` defined in `store/v2alpha1/multi/store.go`).

Reply:
Yes yes. I need to clean that up.